scripts/partition_manager/partition_manager.rst

```diff
@@ -14,6 +14,9 @@ See :ref:`ug_multi_image` for more information about multi-image builds.
 
 The Partition Manager is activated for all multi-image builds, regardless of which build strategy is used for the child image.
 
+.. note::
+   The Device Tree Source flash partitions are ignored when a multi-image application is built using the Partition Manager.
```

Comment:
This is very passive. Could it be a bit more active, for example like this: ... when you build a multi-image application...